packages/node/test/integration/secure-deposit.spec.ts

```diff
@@ -1,7 +1,11 @@
-import { One } from "ethers/constants";
+import DolphinCoin from "@counterfactual/contracts/build/DolphinCoin.json";
+import { ContractABI } from "@counterfactual/types";
+import { Contract, Wallet } from "ethers";
+import { One, Two, Zero } from "ethers/constants";
 import { JsonRpcProvider } from "ethers/providers";
+import { BigNumber } from "ethers/utils";
 
-import { Node } from "../../src";
+import { INSUFFICIENT_ERC20_FUNDS, Node } from "../../src";
```

Comment:
Why is this error exported from this path

Reply:
The errors also get exported by `index.ts` in `/src`